pkgs/development/libraries/libagar/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, pkgconfig, libtool, perl, bsdbuild, gettext, mandoc
+, libpng, libjpeg, xlibsWrapper, libXinerama, freetype, SDL, mesa
+, libsndfile, portaudio, mysql, fontconfig
+}:
+
+let
+  srcs = import ./srcs.nix { inherit fetchurl; };
+in
+stdenv.mkDerivation rec {
+  name = "libagar-${version}";
+  version = srcs.version;
```

Comment:
`inherit (srcs) version src` looks cleaner. Then you could also inline the `import`